subsys/bluetooth/services/bas.c

```diff
@@ -47,6 +47,14 @@ static ssize_t read_blvl(struct bt_conn *conn,
 				 sizeof(lvl8));
 }
 
+static struct bt_gatt_cpf level_cpf = (struct bt_gatt_cpf){
+	.format = 0x04,        /* uint8 */
+	.exponent = 0x0,
+	.unit = 0x27AD,        /* Percentage */
+	.name_space = 0x01,    /* Bluetooth SIG */
+	.description = 0x0106, /* "main" */
```

Comment:
I'm generally not in favor of magic numbers like all these are. 

AFAIK we do not them defined anywhere, so for this it seems to be OK, but we could defining somewhere if they aren't already. 

I guess most of them are from assigned numbers? Maybe a comment on where to find the definitions (if we do not define here as part of this PR) would be useful.

Reply:
Yeah, I tried searching for anywhere there might be defines for these and came up short. They're from the Assigned Numbers database/spec: https://www.bluetooth.com/wp-content/uploads/Files/Specification/Assigned_Numbers.pdf?id=89